src/main/java/dev/seniorjava/githubstatisticsbot/stats/collectors/ReactionStatistics.java

```diff
@@ -0,0 +1,68 @@
+package dev.seniorjava.githubstatisticsbot.stats.collectors;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import dev.seniorjava.githubstatisticsbot.stats.mappers.StatisticsDeserializer;
+import lombok.AllArgsConstructor;
+
+@AllArgsConstructor
+@JsonDeserialize(using = StatisticsDeserializer.class)
+public class ReactionStatistics {
+
+  private User user;
+  private Reactions reactions;
+
+  public String username() {
```

Comment:
Basically, you are right. However, I chose this approach because this is a kind of a value object. As a result, it has no setters and I decided to return its values without the `get` prefix. this makes the code more cleaner. But I admit that someone may find this approach not quite expected.